src/main/java/com/stablekernel/standardlib/RxFragment.java

```diff
@@ -28,6 +28,12 @@ protected void showBlockingProgress() {
         showBlockingProgress(null);
     }
 
+    protected void showBlockingProgress(boolean isCancelable) {
+        blockingProgressFragment = BlockingProgressFragment.newInstance();
+        blockingProgressFragment.show(getFragmentManager(), BlockingProgressFragment.TAG);
+        blockingProgressFragment.setIsCancelable(isCancelable);
+    }
```

Comment:
this should also call into the `protected void showBlockingProgress(Subscription subscription)` method so you don't have duplicated implementations.

you'll probably end up with these 4 method signatures:
```java
protected void showBlockingProgress()
protected void showBlockingProgress(boolean isCancelable)
protected void showBlockingProgress(Subscription subscription)
protected void showBlockingProgress(Subscription subscription, boolean isCancelable)
```